pg_spot_operator/cli.py

```diff
@@ -217,6 +224,16 @@ def compile_manifest_from_cmdline_params(
     m.postgresql.admin_user_password = args.admin_user_password
     m.postgresql.admin_is_superuser = str_to_bool(args.admin_is_superuser)
     m.postgresql.tuning_profile = args.tuning_profile
+    if args.shared_preload_libraries:
+        m.postgresql.config_lines.append(
+            "shared_preload_libraries = '"
+            + args.shared_preload_libraries.rstrip("'").lstrip("'")
+            + "'"
+        )
+    if args.extensions:
+        m.postgresql.extensions = args.extensions.strip().split(",")
+    if args.os_extra_packages:
+        m.os.extra_packages = args.os_extra_packages.strip().split(",")
```

Comment:
TBH we are reaching the limits of CLI; such configs are best put nicely in the manifest:
```
parquet_discovery:
  minimum_time_between_updates: ...
  fdw_sorted_columns: ...
  worker_dbname: ...
pg_stat_statements:
  save: ...
  max: ...
  track: ...
  track_utility: ...
```